src/features/resume.ts

```diff
@@ -76,8 +76,12 @@ export const resumeResources = async (bot: Client) => {
       deferred.edit("Looking for a resume…");
       const messages = await interaction.channel.messages.fetch();
 
-      const { fetchStarterMessage } = interaction.channel;
-      const firstMessage = await retry(() => fetchStarterMessage(), 5, 10);
+      const channel = interaction.channel;
+      const firstMessage = await retry(
+        () => channel.fetchStarterMessage(),
```

Comment:
If we couldn't deconstruct `channel` from `interaction.channel` before because it can be falsy, you might want to nullcheck for `channel` here just to be sure?

Reply:
The underlying issue is not about the value being `null` other than when you have implementation relying on `this` it will lose the context when destructed. For example,
```js
class Foo {
  parent = ""
  
  getParent() {
    return this.parent;
  }
}

const foo = new Foo();
const { getParent } = foo;
getParent() // Uncaught TypeError: Cannot read properties of undefined (reading 'parent')
```